packages/react-storage/src/components/StorageBrowser/views/LocationActionView/CopyView/CopyViewProvider.tsx

```diff
@@ -16,6 +16,7 @@ export function CopyViewProvider({
   children,
   ...props
 }: CopyViewProviderProps): React.JSX.Element {
+  const { CopyView: displayText } = useDisplayText();
```

Comment:
I think the nested destructuring for this should be fine? I.e.
```ts
const {
  CopyView: {
    actionCancelLabel,
    ...
  }
} = useDisplayText();
```

But I see that you're purposefully undoing it in other places - any particular reason why? I don't feel strongly either way, just wondering if we ran into an issue.

Reply:
Nevermind I think I see it's because you want to feed the entire thing back into the table data function